packages/react-charting/src/utilities/utilities.ts

```diff
@@ -1351,7 +1351,7 @@ export const getAccessibleDataObject = (
 
 type LocaleStringDataProps = number | string | Date | undefined;
 export const convertToLocaleString = (data: LocaleStringDataProps, culture?: string): LocaleStringDataProps => {
-  if (!data) {
+  if (data === undefined || data === null || Number.isNaN(data)) {
     return data;
```

Comment:
does isNan return true for string data also

Reply:
Returns False for data types other than Number